Rollbar/NSJSONSerialization+Rollbar.m

```diff
@@ -26,14 +29,14 @@ + (nullable NSData *)dataWithJSONObject:(id)obj options:(NSJSONWritingOptions)op
                 }
             }
             return [NSJSONSerialization dataWithJSONObject:newArr options:opt error:error];
-        } else if ([obj isKindOfClass:[NSDictionary class]]) {
+        } else if ([obj isKindOfClass:[NSMutableDictionary class]]) {
```

Comment:
There is no reason to change this here.

Reply:
yes, reverting it back. as i mentioned in some other comment, i did go paranoid on the mutability of payload components and went with codebase wide almost like "search and replace" :)